ServerCore/Pages/Resources/PH20/FAQPartial.cshtml

```diff
@@ -46,7 +46,7 @@
     <p>At the very least, you should reserve a conference room on the Redmond campus as your Team's HQ and register this location on this web site. Teams lacking appropriate contact and HQ information will not be allowed to participate. </p>
     <p>Keeping your team organized is important both to having fun and doing well. It's a good idea to make sure that the members of your team understand and agree to the levels of commitment that everyone is making. It can cause problems if some people are committed to work through the night and others are not, especially if not everyone understands and agrees to this in advance. Team dynamics generally work better if all team members have similar levels of commitment. </p>
     <h4>Does everyone need to register? </h4>
-    <p>Individual and Team registration is mandatory. @Model.Event.Name and Microsoft Security need to be able to contact all teams at all times. Also, advance registration allows us to ensure team sign-in goes smoothly and the hunt begins on time. Any Microsoft employee may create a team. Simply login, mark yourself as ‘Team Captain’, and send out invites. </p>
+    <p>Individual and Team registration is mandatory. @Model.Event.Name and Microsoft Security need to be able to contact all teams at all times. Also, advance registration allows us to ensure team sign-in goes smoothly and the hunt begins on time. Anyone may create a team. Simply login and create a team, and copy the invite link and mail to your teammates. You can also accept applications to join your team. If you don't want to make a team, you can apply to join someone else's team, but we recommend finding a team in advance with people you know, <a href="mailto:puzzmate@microsoft.com">puzzmate</a>, or the Seattle Puzzlers group on Facebook. You can only apply to one team at a time.</p>
```

Comment:
Should we say something somewhere about needing to check if people have applied and accept them?

Reply:
Reworded a bit to clarify. Thanks!